### DISCUSSION TITLE
Change RabbitType´s

### DISCUSSION THREAD

**Discussion by exfy** [Minestom/Minestom#702] [Help] [answered]
Hello,

I would like to change the "RabbitType" for a rabbit, like "Brown,white etc." Unfortunately it does not work.
I tried the following to spawn a black rabbit: https://minecraft.fandom.com/wiki/Rabbit
`        rabbit.getEntityMeta().setNotifyAboutChanges(true);`
`        rabbit.setTag(Tag.Integer("RabbitType"),2);`
 `       rabbit.spawn();`
How to inform the client about this change? Is there something similar to "getBlockEntityTags()" for blocks to send the changes to the client?
Thanks for the help.

**Comment by MrlnHi** [comment, marked as answer]
```java
((RabbitMeta) entity.getEntityMeta()).setType(RabbitMeta.Type.BLACK)
```
